sdk/rust/oak/src/grpc/mod.rs

```diff
@@ -37,6 +37,21 @@ pub mod server;
 pub type Result<T> = std::result::Result<T, rpc::Status>;
 pub type Invocation = crate::proto::oak::invocation::GrpcInvocation;
 
+impl Invocation {
+    pub fn close(&self) {
+        self.receiver
+            .as_ref()
```

Comment:
Why are all the `as_ref()` necessary? I'm not very familiar with it.

Reply:
Fixed.